### PR TITLE
Make footer mobile responsive

### PR DESCRIPTION
**Pull request by lisazhao30**
**Changes**

- Made footer responsive for mobile
- Made footer responsive for tablet
- I added a second commit to get rid of the redundant css

**How Has This Been Tested?**

- [x] Verified that the design matches the figma
- [x] Tested responsiveness in mobile and tablet mode

See screenshots below for what it looks like on mobile and tablet.
Ipad Air:
Screen Shot 2022-03-12 at 8 59 26 PM
Ipad Pro:
Screen Shot 2022-03-12 at 8 59 11 PM
Smaller Iphone:
Screen Shot 2022-03-12 at 8 43 43 PM
Bigger Iphone
Screen Shot 2022-03-12 at 8 43 52 PM

### CODE REVIEW

**Comment by jennchenn** [comment]
Task linked: [CU-233uwjv Make footer mobile responsive](https://app.clickup.com/t/233uwjv)

**Review by eevanwong — CHANGES_REQUESTED** [review]
Hey! Everything looks good, but I would like for it to stay a bit more consistent with the figma design that was created for the mobile view:

image

We could center the div that encompasses the social tags, then set a max width on it. With this method, you could also use `space-between` instead of `space-evenly`.

**Comment by lisazhao30** [comment]
> Hey! Everything looks good, but I would like for it to stay a bit more consistent with the figma design that was created for the mobile view:
> 
> image
> 
> We could center the div that encompasses the social tags, then set a max width on it. With this method, you could also use `space-between` instead of `space-evenly`.
> 
> Also, could you make the icons + social bubbles smaller to better follow the figma design?

**Comment by lisazhao30** [comment]
For sure, here is what the new design looks like:
Screen Shot 2022-03-14 at 11 24 58 PM

Screen Shot 2022-03-14 at 11 24 43 PM

**Comment by eevanwong** [comment]
Nope, this looks great! Thanks for adding in the new changes! 

Also, no need to close it once changes are requested, you cannot merge it without approval anyways.

**Comment by lisazhao30** [comment]
Yeah sorry, I accidentally closed it!

File: waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx:41]: Can you add `align-items: center` after this line to make sure that the logo and "Contact Us" section are aligned vertically? This can be kept outside of `@media` since they should be aligned in desktop view as well.

File: waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx:44]: How come `padding` was used here rather than `margin`? Although very similar, `margin` seems more appropriate to me. Also, can the same margin be used for top and bottom, i.e. replacing lines 43 and 44 with `margin: 2vh 0;`?

File: waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx:54]: After this line, can you add `margin-right` so that there's more space between the logo and "Contact Us" section? From what I tested, `margin-right: 20px` seems sufficient, but feel free to try different values. This can also be kept outside of `@media`.

**Review by williamhpark — CHANGES_REQUESTED** [review]
Some very minor suggestions from me, but looks great overall. Thanks Lisa!

File: waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx
Comment (lisazhao30) [reply waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx:41]: By putting `align-items: center` outside @media, it goes off the design for the desktop view I think. For now, I have it set to be inside the @media 
Screen Shot 2022-03-16 at 12 54 09 AM
.

**Comment by lisazhao30** [comment]
Screen Shot 2022-03-16 at 1 08 54 AM
Screen Shot 2022-03-16 at 12 54 09 AM

File: waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx
Comment (williamhpark) [reply waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx:42]: What I was hoping to do by setting `align-items: center` here was to have the logo and contact section aligned vertically, because when I saw it on my laptop they were slightly offset in both desktop and mobile views. I didn't want the horizontal position of the logo+contact on the page changed at all. But looking at your recent changes, it looks like you manually aligned them anyways (seems like the SVG itself has some weird whitespace, so that should be fine). So how it is right now should be fine.

**Review by williamhpark — APPROVED** [review]

**Review by eevanwong — APPROVED** [review]
LGTM! Thanks for helping out the web team!

**Pull request merged**
